add sorting of the channel and history lists

jamaljsr opened the pull request:
Closed #86 

This PR adds the ability to manually sort the channel and history lists by clicking on the column headers.

### Steps to Test
1. Go to the Terminal page
1. Click on the column headers (click twice on the same header to change the direction)
1. Confirm the channels are sorting in the correct order
1. Go to the Loop History page
1. Clicking on the column headers
1. Confirm the loops are sorting in the correct order

guggero reviewed (approved):
LGTM :100: 
I really like the idea of the "reset sorting" icon on the main page. Any reason not to add it to the loop history page as well?

jamaljsr commented:
> Any reason not to add it to the loop history page as well?

The only reason I added the "reset sorting" icon is because the channel list is initially sorted by the Balance Mode defined in Settings and this column has no header to click. If the user sorts manually, they need a way to get back to the original Balance Mode sorting. I initially added a header to this column, but I didn't like the way it looked. So I added this icon instead.

The Loop history list is sorted by Update Date by default, which does have a column header. So the user can easily revert to this sorting method by clicking on the header if they choose to do so.

Hope that makes sense.

guggero commented:
Ah, makes sense, thanks!
